CMakeLists.txt

```diff
@@ -17,17 +16,13 @@ find_package(message_filters REQUIRED)
 find_package(tf2_ros REQUIRED)
 find_package(laser_geometry REQUIRED)
```

Comment:
Eigen3 include dirs. is passed through laser_geometry_INCLUDE_DIRS … that change can be found in this commit:

https://github.com/ros-perception/laser_geometry/pull/27/commits/5cd2f1c9b617c9b42fdfac9f6d54e96e823e2e9a

Reply:
Awesome!